spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/scutilities/WFSUtils.java

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2018 Boundless, http://boundlessgeo.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License
+ */
+package com.boundlessgeo.spatialconnect.scutilities;
+
+import android.util.Log;
+
+import com.boundlessgeo.spatialconnect.geometries.SCGeometry;
+import com.boundlessgeo.spatialconnect.geometries.SCSpatialFeature;
+import com.vividsolutions.jts.geom.Geometry;
+
+import java.util.Map;
+
+public class WFSUtils {
+    private static String LOG_TAG = WFSUtils.class.getSimpleName();
+
+    private WFSUtils(){}
+
+    public static String buildWFSTInsertPayload(SCSpatialFeature scSpatialFeature, String remoteUrl) {
+        // assumes "geonode" will always be the GeoServer workspace for Exchange backend
+        String featureTypeUrl = String.format("%s/geoserver/wfs/DescribeFeatureType?typename=%s:%s",
+                remoteUrl, "geonode", scSpatialFeature.getLayerId());
+
+        return String.format(WFST_INSERT_TEMPLATE,
+                featureTypeUrl,
+                scSpatialFeature.getLayerId(),
+                buildPropertiesXml(scSpatialFeature),
+                buildGeometryXml(scSpatialFeature));
+    }
+
+    public static String buildWFSTUpdatePayload(SCSpatialFeature scSpatialFeature, String remoteUrl) {
+        return String.format(WFST_UPDATE_TEMPLATE,
+                scSpatialFeature.getLayerId(),
+                buildPropertiesXml(scSpatialFeature),
+                buildFilterXml(scSpatialFeature));
+    }
+
+    public static String buildWFSTDeletePayload(SCSpatialFeature scSpatialFeature, String remoteUrl) {
+        return String.format(WFST_DELETE_TEMPLATE,
+                scSpatialFeature.getLayerId(),
+                buildFilterXml(scSpatialFeature));
+    }
+
+    private static String buildFilterXml(SCSpatialFeature feature) {
+        String ogcFilter = "<ogc:Filter>\n"
+                            + "   <ogc:FeatureId fid=\"%1$s\"/>\n"
+                            +"</wfs:Transaction>";
```

Comment:
Should this closing tag be `</ogc:Filter>`?

Reply:
yes, :-)